docs/features/logforwarding/custom-tenants/custom-tenant-support.adoc

```diff
@@ -0,0 +1,246 @@
+= Configuring Custom Tenants
+
+Most of the outputs supported have some concept of a `tenant` and can currently be configured with custom values. For example `Kafka` has `topics`, `CloudWatch` has `log_groups`, and `ElasticSearch` has `indices`. This document provides guidance on configuring the tenant for supported outputs.
+
+== CloudWatch
+CloudWatch's tenant can be configured by setting how logs are grouped together along with an optional prefix. Configure using the `groupBy` & `groupPrefix` fields.
+
+* Logs can be grouped by the following:
+. `namespaceName`
+. `namespaceUUID`
+. `logType`
+
+=== Example
+
+.cluster-log-forwarder.yaml
+[source,yaml]
+----
+spec:
+  outputs:
+   - name: cw 
+     type: cloudwatch 
+     cloudwatch:
+       groupBy: logType <1>
+       groupPrefix: <group prefix> <2> 
+       region: us-east-2
+----
+<1> Specify the `groupBy` type here.
+<2> Specify an optional prefix to add to the tenant.
+
+=== References
+
+. See the link:https://docs.openshift.com/container-platform/4.14/logging/log_collection_forwarding/configuring-log-forwarding.html#cluster-logging-collector-log-forward-cloudwatch_configuring-log-forwarding[forwarding to CloudWatch section] on the official documentation.
+
+== Elasticsearch
+Elasticsearch's default behavior is to sends logs to indices defined by their `log_type` and appended by `-write`.
+
+  Example: `application` logs will be sent to the index `app-write`.
+
+ElasticSearch's index can be configured if `parse: json` is enabled.
+
+* Configure using the `structuredTypeKey & structuredTypeName` fields.
+
+=== Example
+
+.cluster-log-forwarder.yaml
+[source,yaml]
+----
+spec:
+  outputs:
+    - name: external-es
+      type: elasticsearch
+      elasticsearch: 
+        structuredTypeKey: kubernetes.namespace_name <1>
+        structuredTypeName: myParsedMessages <2>
+  pipelines:
+    - name: parsed-app-logs
+      inputRefs:
+        - application
+      outputRefs:
+        - external-es
+      parse: json <3>
+
+----
+<1> Specify the `structuredTypeKey` here.
+<2> Specify the `structuredTypeName` here.
+<3> Parse JSON must be enabled.
+
+NOTE: Both `structuredTypeKey` & `structuredTypeName` must be specified.
```

Comment:
Not technically true.   One or both.  I would additionally (briefly) mention that 'name' acts as a fallback if 'key' is not found.